cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -187,8 +209,13 @@ pub type PoolFungiblesTransactor = FungiblesAdapter<
 >;
 
 /// Means for transacting assets on this chain.
-pub type AssetTransactors =
-	(CurrencyTransactor, FungiblesTransactor, ForeignFungiblesTransactor, PoolFungiblesTransactor);
+pub type AssetTransactors = (
+	CurrencyTransactor,
+	FungiblesTransactor,
+	UniquesTransactor,
```

Comment:
I would maybe put it at the end, if we expect more fungible than non-fungible operations

Reply:
Seems like a realistic expectation. Updated the order.